src/table/pagination-table/components/body/TableBodyWrapper.tsx

```diff
@@ -42,10 +43,11 @@ const TableBodyWrapper = ({ setShouldRefocus, tableWrapperRef, announce }: Table
   const hoverEffect = !!getStylingComponent(layout)?.content?.hoverEffect;
 
   useSelectionListener({ keyboard, selectionDispatch, selectionsAPI, setShouldRefocus, tableWrapperRef });
+  const showTotals = (totalsPosition.atTop || totalsPosition.atBottom)  && viewService.viewState?.showTotals !== false;
```

Comment:
Can you not integrate `viewService.viewState?.showTotals` in to the `getTotalPosition` function instead? Then you only need to resolve it once and both pagination table and virtual scroll table should get it

Reply:
Good suggestion. I will look into this.

Reply:
The idea is that we only show totals row on the first pdf page but not from the second pdf page.